Documentation/policy/intro.rst

```diff
@@ -32,6 +32,11 @@ always
   With always mode, policy enforcement is enabled on all endpoints even if no
   rules select specific endpoints.
 
+  If you want to configure health entity to check cluster-wide connectivity when 
+  you start cilium-agent with ``enable-policy=always``, With that mode 
+  you will likely want to enable communications to and from the health endpoint.
+  See :ref:`health_endpoint`.
```

Comment:
It looks like you copy-pasted Paul's suggestion, please make sure the resulting sentence reads well.

```suggestion
  If you want to configure health entity to check cluster-wide connectivity when 
  you start cilium-agent with ``enable-policy=always``, you will likely want to
  enable communications to and from the health endpoint.
  See :ref:`health_endpoint`.
```